examples/hybrid/types.jl

```diff
@@ -364,25 +372,32 @@ function args_integrator(parsed_args, Y, p, tspan, ode_config, callback)
     dt_save_to_sol = time_to_seconds(parsed_args["dt_save_to_sol"])
     show_progress_bar = isinteractive()
 
-    @time "Define problem" problem = if parsed_args["split_ode"]
-        remaining_func =
-            startswith(parsed_args["ode_algo"], "ODE.") ?
-            remaining_tendency! :
-            ForwardEulerODEFunction(remaining_tendency_increment!)
-        ODE.SplitODEProblem(
-            ODE.ODEFunction(
-                implicit_tendency!;
-                jac_kwargs(ode_algorithm, Y, atmos.energy_form)...,
-                tgrad = (∂Y∂t, Y, p, t) -> (∂Y∂t .= 0),
-            ),
-            remaining_func,
-            Y,
-            tspan,
-            p,
+    @time "Define function" func = if parsed_args["split_ode"]
+        implicit_func = ODE.ODEFunction(
+            implicit_tendency!;
+            jac_kwargs(ode_algorithm, Y, atmos.energy_form)...,
+            tgrad = (∂Y∂t, Y, p, t) -> (∂Y∂t .= 0),
         )
+        if startswith(parsed_args["ode_algo"], "ODE.")
+            ODE.SplitODEFunction(
+                implicit_func,
+                remaining_tendency!, # should be remaining_tendency_plus_dss!
+            )
+        else
+            CTS.ClimaODEFunction(;
+                T_exp! = remaining_tendency!,
+                T_imp! = implicit_func,
+                # Can we just pass implicit_tendency! and jac_prototype etc.?
+                dss!,
```

Comment:
Reminder: comment out the `dss!` in the `remaining_tendency!` and `implicit_func`s